### DISCUSSION TITLE
Welcome to Trading-Agent- Discussions!

### DISCUSSION THREAD

**Discussion by MiChaelinzo** [MiChaelinzo/Trading-Agent-#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by imautogpt** [comment]
Thanks for uploading. Super cool.
Are there any other Machine Learning models people can recommend that can be as good as this one? Or close to.
